src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -274,15 +282,26 @@ public DataAccessRequest createProgressReport(User user, DataAccessRequest progr
   }
 
   public void validateProgressReport(User user, DataAccessRequest progressReport, DataAccessRequest parentDar) {
-    validateDar(user, progressReport);
-    if (parentDar.getDraft()) {
+    validateCommonDarAndProgressReportElements(user, progressReport);
+    List<String> errorSummary = new ArrayList<>();
+    getErrorSummary(progressReport.getData().getInternalCollaborators().stream().map(Collaborator::getEmail).toList(), user.getInstitution(),
+        INTERNAL_COLLABORATOR + " " + MEMBER + ": ", INTERNAL_COLLABORATOR + " " + MEMBERS, errorSummary);
+    getErrorSummary(progressReport.getData().getLabCollaborators().stream().map(Collaborator::getEmail).toList(), user.getInstitution(),
+        LAB_STAFF + " " + MEMBER + ": ", LAB_STAFF + " " + MEMBERS, errorSummary);
+
+    if (!errorSummary.isEmpty()) {
+      throw new BadRequestException( ALL_LISTED_PERSONNEL_MUST_SHARE_THE_SAME_INSTITUTION
+          + String.join(", ", errorSummary));
+    }
+
+    if (Boolean.TRUE.equals(parentDar.getDraft())) {
```

Comment:
This means that if `parentDar.getDraft()` is `null`, it's not a draft. Is that correct? Is there a case where this field is `null`?

Another way to express this is to use `Objects.requireNonNullElse()` which may make the behavior more apparent.
```suggestion
    if (Objects.requireNonNullElse(parentDar.getDraft(), false)) {
```

Reply:
Good point.  I did check DataAccessRequest and we currently establish a value to true on instance creation and only allow it to be mutated to true and false again using internally controlled values that never set this to null.  Additionally, there's no public setter on this value.